client/my-sites/checkout/composite-checkout/composite-checkout.tsx

```diff
@@ -638,6 +644,11 @@ export default function CompositeCheckout( {
 		);
 	}
 
+	const handlePaymentComplete = ( args: PaymentCompleteCallbackArguments ) => {
```

Comment:
Minor, I think, but just in case there's something relying on this function's identity (I'd have to go through the code to be sure but I see that `useCreatePaymentCompleteCallback` memoizes its result), it may be good to wrap this in a `useCallback()` hook with the relevant dependencies so it doesn't change if it doesn't need to.

Reply:
That's a good point. Updated in https://github.com/Automattic/wp-calypso/pull/48452/commits/81724b80122025547fbbb007762d2a133cc95cf1